src/Sylius/Behat/Context/Api/Shop/AddressContext.php

```
@@ -375,26 +304,20 @@ public function iShouldBeNotifiedThatTheProvinceNeedsToBeSpecified(): void
     public function iShouldStillBeOnTheAddressEditPage(string $fullName): void
     {
         // Intentionally left empty
     }
 
-    /**
-     * @Then I should still have :provinceName as my specified province
-     * @Then I should still have :provinceName as my chosen province
-     */
+    #[Then('I should still have :provinceName as my specified province')]
+    #[Then('I should still have :provinceName as my chosen province')]
     public function iShouldStillHaveAsMySpecifiedProvince(string $provinceName): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$provinceName'. (undefined)

(UnusedFormalParameter)